src/Migrations/M040ReplaceSpecialCharactersInPostsContent.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+use WP_Query;
+
+/**
+ * Replace special characters ("u003c", "u003e", "u0022") in the posts content.
+ */
+class M040ReplaceSpecialCharactersInPostsContent extends MigrationScript
+{
+    /**
+     * Perform the actual migration.
+     *
+     * @param MigrationRecord $record Information on the execution, can be used to add logs.
+     * phpcs:disable SlevomatCodingStandard.Functions.UnusedParameter -- interface implementation
+     */
+    public static function execute(MigrationRecord $record): void
+    {
+        echo 'Replacing special characters in posts content...', "\n"; // phpcs:ignore
+
+        $post_types = array_diff(get_post_types(['public' => true], 'names'), ['archive']);
+        $paged = 1;
+        $per_page = 30;
+
+        do {
+            $query = new WP_Query([
+                'post_type' => $post_types,
+                'posts_per_page' => $per_page,
+                'paged' => $paged,
+                'post_status' => 'any',
+                'fields' => 'ids',
+            ]);
+
+            if (!$query->have_posts()) {
+                break;
+            }
+
+            foreach ($query->posts as $post_id) {
+                $content = get_post_field('post_content', $post_id);
+
+                if (empty($content)) {
+                    continue;
+                }
+
+                echo 'Parsing post ', $post_id, "\n"; // phpcs:ignore
+
+                $updated_content = preg_replace('/(?<!\\\\)u003c/', '<', $content);
+                $updated_content = preg_replace('/(?<!\\\\)u003e/', '>', $updated_content);
+                $updated_content = preg_replace('/(?<!\\\\)u0022/', '"', $updated_content);
+
+                if ($updated_content === $content) {
+                    continue;
+                }
+
+                wp_update_post([
```

Comment:
Can we print the update status, just in case if we want to check which posts are updated in migration?